libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/NavigationAlertView.java

```diff
@@ -0,0 +1,123 @@
+package com.mapbox.services.android.navigation.ui.v5.instruction;
+
+import android.content.Context;
+import android.os.Handler;
+import android.support.annotation.Nullable;
+import android.support.v4.app.FragmentActivity;
+import android.support.v4.app.FragmentManager;
+import android.util.AttributeSet;
+import android.view.View;
+
+import com.mapbox.services.android.navigation.ui.v5.NavigationViewModel;
+import com.mapbox.services.android.navigation.ui.v5.alert.AlertView;
+import com.mapbox.services.android.navigation.ui.v5.feedback.FeedbackBottomSheet;
+import com.mapbox.services.android.navigation.ui.v5.feedback.FeedbackBottomSheetListener;
+import com.mapbox.services.android.navigation.ui.v5.feedback.FeedbackItem;
+import com.mapbox.services.android.navigation.v5.navigation.NavigationConstants;
+import com.mapbox.services.android.navigation.v5.navigation.metrics.FeedbackEvent;
+
+import timber.log.Timber;
+
+public class NavigationAlertView extends AlertView implements FeedbackBottomSheetListener {
+  private NavigationViewModel navigationViewModel;
+
+  public NavigationAlertView(Context context) {
+    this(context, null);
+  }
+
+  public NavigationAlertView(Context context, @Nullable AttributeSet attrs) {
+    this(context, attrs, -1);
+  }
+
+  public NavigationAlertView(Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
+    super(context, attrs, defStyleAttr);
+  }
+
+  /**
+   * Sets the NavigationViewModel in the view
+   *
+   * @param navigationViewModel to set
+   */
+  public void setModel(NavigationViewModel navigationViewModel) {
```

Comment:
To align method naming here, does it make sense to update this to `subscribe` like `InstructionView#subscribe(NavigationViewModel)`?